crates/actor/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "wasmcloud-actor"
+version = "0.1.0"
+description = "wasmCloud actor functionality"
```

Comment:
Not needed here, but a more robust description for this would be nice

Reply:
Yeah, this is mostly just a placeholder. Not sure what this will look like yet.

I'll see how to improve it